### PR TITLE
SLLS-107 Enable analysis of CSS code

### PR DESCRIPTION
**Pull request by jblievremont**

### CODE REVIEW

**Review by sophio-japharidze-sonarsource — APPROVED** [review]
LGTM! 👍
But maybe we should still wait for the SonarJS release before merging?

**Comment by jblievremont** [comment]
IMHO, since we don't plan to release publicly at the end of the sprint, we can merge and send this to dogfood, and we'll bump to a released version when the time comes.

**Pull request merged**
